api/CHANGELOG.md

```diff
@@ -1 +1,3 @@
-## Spree 2.4.0 (unreleased) ##
+## Spree 3.0.0 (unreleased) ##
```

Comment:
I think we should remove the changelog files, and just keep track of changes in the guides release notes of each version.